canvas_modules/common-canvas/src/common-properties/controls/passwordfield/passwordfield.jsx

```diff
@@ -34,6 +35,14 @@ class PasswordControl extends React.Component {
 	}
 
 	render() {
+		const overrideLabelKeyShow = `${this.props.control.name}.passwordShow.tooltip`;
+		const overrideLabelKeyHide = `${this.props.control.name}.passwordHide.tooltip`;
+
+		const defaultShowEditLabel = formatMessage(this.reactIntl, MESSAGE_KEYS.SHOW_PASSWORD_TOOLTIP);
+		const defaultHideEditLabel = formatMessage(this.reactIntl, MESSAGE_KEYS.HIDE_PASSWORD_TOOLTIP);
```

Comment:
`defaultShowEditLabel` can be renamed to `defaultShowPasswordLabel`
`defaultHideEditLabel` can be renamed to `defaultHidePasswordLabel`